packages/dds/tree/src/feature-libraries/modular-schema/genericFieldKind.ts

```diff
@@ -72,14 +71,11 @@ export const genericChangeHandler: FieldChangeHandler<GenericChangeset> = {
 			}
 			return composed;
 		},
-		invert: (
-			{ change }: TaggedChange<GenericChangeset>,
-			invertChild: NodeChangeInverter,
-		): GenericChangeset => {
+		invert: ({ change }: TaggedChange<GenericChangeset>): GenericChangeset => {
 			return change.map(
```

Comment:
Does the MCF contract require this shallow copy? Can we not just return `change` as is?